runtime/help/commands.md

```diff
@@ -72,12 +72,11 @@ quotes here but these are not necessary when entering the command in micro.
 * `setlocal 'option' 'value'`: sets the option to value locally (only in the
    current buffer). This will *not* modify `settings.json`.
    
-* `toggle 'option'`: toggles the option to `on/off`. See the `options` help
-  topic for a list of options you can toggle, only works with `on/off` options. This will modify your
+* `toggle 'option'`: toggles the option `true/false`. Only works with boolean options. This will modify your
```

Comment:
Again, why not just "toggles the option on/off" (without backquotes), i.e. just use human language here?